icons/pipedream.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Pipedream</title><path d="M12 0C.213 0 .16 0 .08.08 0 .16 0 .213 0 12c0 11.787 0 11.84.08 11.92C.16 24 .213 24 12 24c11.787 0 11.84 0 11.92-.08.08-.08.08-.133.08-11.92C24 .213 24 .16 23.92.08 23.84 0 23.787 0 12 0Zm.942 3.493a6.031 6.031 0 0 1 1.629.183c1.952.493 2.98 1.955 3.336 4.744.1.787.102 3.024.003 3.71-.255 1.765-.797 3.037-1.668 3.914-.674.677-1.427 1.055-2.436 1.22-.572.094-1.638.057-2.209-.077-.654-.154-1.098-.381-1.527-.78-.105-.098-.192-.162-.194-.142-.003.019-.001.985.004 2.145l.008 2.11H8.203c-1.312 0-1.69-.011-1.704-.05-.01-.028-.015-3.785-.01-8.35l.011-8.3 1-.067c.55-.037 1.081-.068 1.18-.07l.18-.003.317.576.317.576.27-.218c.873-.705 1.996-1.091 3.178-1.121Zm-.937 3.076a3.1 3.1 0 0 0-.748.103c-.42.132-.811.365-1.105.658l-.272.273v5.75l.154.195c.34.43.858.767 1.337.872.331.072 1.023.072 1.36.001.952-.202 1.497-1.01 1.714-2.541.072-.506.074-2.487.003-2.98-.215-1.497-.634-2.077-1.648-2.282a3.652 3.652 0 0 0-.795-.049z"/></svg>
```

Comment:
I'm seeing differences on all 4 corners:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/6fe41251-f580-4e93-9d4b-610f2648eea7)

It might be better to recreate the square with slight radius and apply the `p` from the website header SVG. Can you give this another look?

Reply:
Sure, I'll look into it!

Reply:
Do you still want to give this a try @aargeee?